app/form_models/school_search_form.rb

```diff
@@ -47,8 +47,8 @@ def job_availability_options
 
   def organisation_type_options
     [
-      [I18n.t("helpers.label.publishers_job_listing_working_patterns_form.organisation_type_options.academy"), "includes free schools"],
-      [I18n.t("helpers.label.publishers_job_listing_working_patterns_form.organisation_type_options.local_authority"), nil],
+      [I18n.t("helpers.label.publishers_job_listing_contract_information_form.organisation_type_options.academy"), "includes free schools"],
+      [I18n.t("helpers.label.publishers_job_listing_contract_information_form.organisation_type_options.local_authority"), nil],
```

Comment:
Just an observation, as this is not related to your changes.  
I suspect these translations were originally defined in the working patterns form, but the fields were moved later to a different step/form, while the translations where left behind!